src/componenets/teaModal/index.jsx

```diff
@@ -1,28 +1,109 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+import {
+  createOrder,
+  deleteOrders,
+  updateOrders,
+} from "../../services/dataServices";
 import {
   Box,
   Container,
   Card,
   Typography,
   Divider,
-  InputLabel,
   MenuItem,
   Select,
+  Grid,
 } from "@mui/material";
 import CustomButton from "../../common/button";
 import CustomTextField from "../../common/textField";
+import { useDispatch, useSelector } from "react-redux";
+import { addTea, deleteTea, updateTea } from "../../redux/actions/teaAction";
+import Header from "../header";
+import toast from "react-hot-toast";
+import { Toaster } from "react-hot-toast";
+import { getEmployeeOrder } from "../../services/dataServices";
 
 const TeaModal = () => {
+  const [alldata, setAllData] = useState();
+  const user = useSelector(
+    (state) => state?.user?.data?.data?.payload?.data?.user
+  );
```

Comment:
please send only required data from api call and what if there is no user it will undefined